utils/grafana.js

```diff
@@ -0,0 +1,81 @@
+import { parse as parseUrl, addParam } from '@/utils/url';
+
+export function computeDashboardUrl(embedUrl, clusterId, params) {

```

Comment:
Observation: I think there's already a well established pattern for independently exported functions like this file, but I think it would be good in the future to switch to something more module like. We'd need consensus from all devs though

```
var double = function(x) {
  return x * 2;
}
export { double };
```